### PR TITLE
Add image location and cleanup configmap

### PR DESCRIPTION
**Pull request by yasensim**
@salv-orlando can I ask you to review this pr?

### CODE REVIEW

**Comment by yasensim** [comment]
@dantingl  could you also review it please?

**Comment by yasensim** [comment]
The idea of this pr is to cleanup parameters that are not relevant to Openshift in the configmap

File: deploy/configmap.yaml [hunk 1]
```diff
@@ -8,23 +8,13 @@ data:
 
     [vc]
```
Comment (dantingl) [inline deploy/configmap.yaml:9]: We can remove the section header [vc] as well

File: deploy/configmap.yaml [hunk 2]
```diff
@@ -540,44 +468,23 @@ data:
...
-
-
-
-    #[nsx_v3]
-    # Deprecated option: tier0_router
-    # Replaced by [nsx_v3] top_tier_router
-
-    # Deprecated option: deny_subnets_redistribution
-    # Replaced by [nsx_v3] configure_t0_redistribution
+    process_oc_network = False
```
Comment (salv-orlando) [inline deploy/configmap.yaml:478]: We should be default to False here in the operand, aren't we?

File: deploy/operator.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ spec:
       containers:
         - name: nsx-ncp-operator
           # Replace this with the built image name
-          image: REPLACE_IMAGE
+          image: vmware/nsx-container-plugin-operator:0.0.3
```
Comment (salv-orlando) [inline deploy/operator.yaml:26]: We have now 0.0.4 and soon 0.0.5, perhaps we should update this PR

File: deploy/configmap.yaml [hunk 4]
```diff
@@ -87,18 +73,11 @@ data:
     # Total number of compressed backup files to store. Defaults to 5.
     #log_rotation_backup_count = 5
 
-
-
-
     [nsx_v3]
```
Comment (salv-orlando) [inline deploy/configmap.yaml:76]: I think there are some topology related options that can be dropped

**Comment by salv-orlando** [comment]
I believe this PR is now superseded, therefore closing it. Reopen it if necessary.

**Pull request closed without merging**